Biomit_workshop_3_Reporting with R markdown.Rmd

```diff
@@ -342,11 +342,13 @@ Here is where it get fancy....In the YAML header you can specify the custom para
 
 library(dplyr)
 reproducible_tab <- NHANES %>% filter(Education == "params$education") %>% 
-                      group_by(params$group_var) %>% 
+                      group_by(.data[[params$group_var]]) %>% 
```

Comment:
Fro the Vignette on tidyverse: ""When you have an env-variable that is a character vector, you need to index into the .data pronoun with [[, like summarise(df, mean = mean(.data[[var]])).  "  So does it mean YAML params are evaluated as strings under the hood?